modules/shortcodes/getty.php

```diff
@@ -146,8 +152,10 @@ function wpcom_shortcodereverse_getty( $content ) {
 			if ( ! empty( $height ) ) {
 				$shortcode .= ' height="' . esc_attr( $height ) . '"';
 			}
-			// While it does not appear to have any practical impact, Getty has
-			// requested that we include TLD in the embed request
+			/**
```

Comment:
I don't think this should be a doc block. Should it?

```suggestion
			/*
```